lago/virt.py

```diff
@@ -332,14 +332,29 @@ def start(self, vm_names=None):
                     rollback.prependDefer(vm.stop)
                 rollback.clear()
 
-    def stop(self, vm_names=None):
+    def _get_stoppable_vms_and_nets(self, vm_names):
```

Comment:
I'm not sure if I wouldn't have done get_stoppable_vms() and get_stoppable_nets()

Also, the word 'stoppable' is a bit strange. _get_running_nets() and _get_running_vms() perhaps?